template-params.yaml

```
@@ -0,0 +1,7 @@
+image:
```

The contents of this file are unused so I think you can delete everything here and add this:
```suggestion
gke_project: SET_PROJECT_ID
```
and then delete `gke_project` from all the test files in `lwt/*.yaml`.

---

Thanks - Addressed [here](https://github.com/datastax/fallout-tests/pull/1/commits/65a8b13f5e50729b143a9e3db55fe0dfccdbf035) and [here](https://github.com/datastax/fallout-tests/pull/1/commits/2877d57078c16b29dc8f60a439f61e44b37caf07)